packages/web/package.json

```diff
@@ -19,6 +19,8 @@
     "@chakra-ui/react": "^1.6.7",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
+    "axios": "^0.23.0",
```

Comment:
We should use `fetch` instead, it's already in all browsers so we don't need to add a library

```suggestion
```